Assignments/stu8/data-access.js

```diff
@@ -8,7 +8,7 @@ const GET_ACCOUNT_TYPE = `select * from account_type`
 
 const GET_TRANSACTION_TYPE = `select * from transaction_type`
 
-
+const GET_USER_BY_DOB = `select * from bank_user where dob > '12-31-1970'`
```

Comment:
We can't hardcode this date here.

Reply:
Ack.  No wonder it felt so freaking easy.